packages/retail-ui/styleguide/PathlineRenderer.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Styled from 'rsg-components/Styled';
+
+const styles = ({ font, base, light, link, baseBackground, mq }) => ({
+  root: {
+    color: base,
+    backgroundColor: baseBackground,
+  },
+  content: {
+    fontFamily: font,
+    paddingTop: '8px',
+    paddingBottom: '8px',
+  },
+});
+
+const ALL_ISSUES_LINK = 'https://github.com/skbkontur/retail-ui/issues';
+const API_URL = 'https://guides.kontur.ru/github';
+const GUIDES_LINK = 'https://guides.kontur.ru/components/';
+
+export class PathlineRenderer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      issueList: [],
+      isFetching: true,
+      hasGuidesLink: false,
+      componentInGuideUrl: '',
+    };
+  }
+
+  componentDidMount = async () => {
+    try {
+      const component = this.getComponentName().toLowerCase();
+
+      this.toggleFetchingState(true);
+      await this.getIssueList(component);
+      await this.getGuidesLink(component);
+      this.toggleFetchingState(false);
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  getGuidesLink = async (component) => {
```

Comment:
```suggestion
  checkGuideForComponent = async (component) => {
```
И наверно лучше хранить в стейте флаг, а не урл